chain/bitcoind_conn.go

```diff
@@ -147,9 +147,29 @@ func NewBitcoindConn(cfg *BitcoindConfig) (*BitcoindConn, error) {
 	}
 
 	// Verify that the node is running on the expected network.
-	net, err := getCurrentNet(client)
-	if err != nil {
-		return nil, err
+	var (
+		net wire.BitcoinNet
+	)
+	btcInWarmup := false
+	for {
+		net, err = getCurrentNet(client)
+		if err == nil {
+			// No error means bitcoind is responsive and we can proceed.
+			break
+		} else if rpcErr, ok := err.(*btcjson.RPCError); ok && rpcErr.Code == btcjson.ErrRPCInWarmup {
```

Comment:
Nit: line exceeds 80 character limit.

Reply:
If I split this into two lines, for instance by moving everything after the semicolon to its own line, then `make lint` doesn't pass and tells me to run `gofmt`, but if I do that it moves this `else if` statement back to one line. So I'm not sure what to do, it feels like I'm fighting against the style rules defined by the project.

Reply:
So the `else if` can just become an `if` since the clause above breaks out. We can then split the line at `&&`.

Reply:
Thank you for that bit of hand holding, that works out.